views.py

```diff
@@ -15,38 +15,38 @@
 
 def split_races(results):
 
-	races = []
+    races = []
 
-	for x in range(1, 8):
+    for x in range(1, 8):
 
-		race = []
+        race = []
 
-		for result in results:
-			if result[2] is x:
-				race.append(result)
+        for result in results:
+            if result[2] is x:
+                race.append(result)
 
-		races.append(race)
+        races.append(race)
 
-	return races
+    return races
 
 
 # ----- Routes ----- #
 
 @app.route('/2014')
 def index():
 
-	query = 'SELECT * FROM results where year > 2014'
-	results = db.fetchall(query)
-	races = split_races(results)
+    query = 'SELECT * FROM results where year > 2014'
```

Comment:
`where` -> `WHERE`

Obvs not from this PR, in fact I messed this up in c55a4d27, but we should tidy this up at some point 🙂.